string get_msg
uint8 get_val
---
string echo_msg
uint8 echo_val
